modules/bmo-configuring-networkdata.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-configuring-networkdata_{context}"]
+= Configuring networkData
+
+The `networkData` field in the `BareMetalHost` custom resource (CR) allows you to control the network configuration of the bare-metal host at creation time. For most operating systems, this is achieved using a configuration file encapsulated in a Kubernetes secret. Then, the `cloud-init` service uses it to customize services. 
+
+[NOTE]
+====
+For the Fedora CoreOS operating system, use Ignition instead.
```

Comment:
we could provide a link to Fedora CoreOS doco again: https://docs.fedoraproject.org/en-US/fedora-coreos/sysconfig-network-configuration/#_via_ignition

this is a bit raw but better than nothing. 

CoreOSes are IMHO a bit of a pain with network configuration. I think they are supposed to use NMstate but official doco just says to inject network configuration using ignition instead of using a mechanism similar to networkData.

Since this is BMaaS doco and not CoreOS I am tempted to provide the best resource I have (which is link above) and let the user read up as much as required for their specific use case.

Reply:
Added.